dependabot-combine.sh

```diff
@@ -2,40 +2,89 @@
 
 set -e # The '-e' option causes bash to exit on errors
```

Comment:
What are you're thoughts on removing this so the error is presumably displayed instead of just disappearing?

Reply:
My understanding is that it's a best practice to always include this in a bash script. It shouldn't have any effect on the output, but if a command fails it will stop the script from continuing in a bad state.